plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/impl/LogPropertyPropertiesEditionPartImpl.java

```diff
@@ -507,32 +508,45 @@ protected Composite createPropertyExpressionText(final Composite parent) {
 		propertyExpressionTypeElements = new Control[] { propertyExpressionTextLabel, propertyExpressionText,
 				propertyExpressionHelp };// mouse
 		propertyExpressionText.addMouseListener(new MouseListener() {
+		    
+            @Override
+            public void mouseDown(MouseEvent e) {
+                openPropertyExpressionNSPEDialog(parent);
+            }		    
 
 			@Override
-			public void mouseUp(MouseEvent e) {
-				// TODO Auto-generated method stub
-
-			}
-
-			@Override
-			public void mouseDown(MouseEvent e) {
-			    EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(), SWT.NULL, propertyExpression);
-                nspd.open();
-                propertyExpressionText.setText(propertyExpression.getPropertyValue());
-                propertiesEditionComponent.firePropertiesChanged(new PropertiesEditionEvent(LogPropertyPropertiesEditionPartImpl.this, 
-                        EsbViewsRepository.LogProperty.Properties.propertyExpression, PropertiesEditionEvent.COMMIT, PropertiesEditionEvent.SET,
-                        null, getExpressionValue()));
-                
-			}
+			public void mouseUp(MouseEvent e) {}
 
 			@Override
-			public void mouseDoubleClick(MouseEvent e) {
-				// TODO Auto-generated method stub
-
-			}
+			public void mouseDoubleClick(MouseEvent e) {}
+			
 		});
+		
+		propertyExpressionText.addKeyListener(new KeyListener() {
+            
+            @Override
```

Comment:
Fix formatting issue.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804